FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -82,6 +91,12 @@ typedef void (^FIRRemoteConfigFetchCompletion)(FIRRemoteConfigFetchStatus status
                                                NSError *_Nullable error)
     NS_SWIFT_UNAVAILABLE("Use Swift's closure syntax instead.");
 
+/// Completion handler invoked by config update methods when they get a response from the server.
+///
+/// @param error  Error message on failure.
+typedef void (^FIRConfigUpdateCompletion)(NSError *_Nullable error)
```

Comment:
I don't see addOnConfigUpdateListener in the public API as the API review suggested. Is this still working progress?

Reply:
Yeah, I wanted to create the client class first and fill out some of the logic before I exposed the public API in FIRRemoteConfig. Thanks!

Reply:
If possible, please avoid adding typedefs to the public API.  typedef's don't port to Swift and we don't want to increase the tech debt for a Swift transition.

Reply:
agree, you can simply make it a callback with NSError parameter in it like this one:
https://github.com/firebase/firebase-ios-sdk/blob/eb30b6fdd809cbfe3a9eb7f3ccf1dd46326bef2d/FirebaseAppDistribution/Tests/Unit/FIRAppDistributionTests.m#L39

Reply:
I updated this already to just a callback as described above